scss/bitstyles/ui/notifications.stories.mdx

```diff
@@ -65,23 +65,20 @@ Some notifications may require or enable further actions by the user — add but
     {`
      <div style="position: relative; z-index:0; transform: translate3d(0,0,0); min-height: 15rem">
         <div class="o-notification-center a-content a-content--s u-margin-0" aria-live="polite">
-          <article class="a-card u-padding-l2 u-flex u-items-start">
-            <div class="u-flex-shrink-0 a-badge a-badge--positive u-padding-s2 u-margin-m-right">
+          <article class="u-flex a-card u-border-radius-0">
```

Comment:
Let’s update this mdx to match the format we’re using for the other new components (which will make this ready for storybook v7 too) — all the stories live in the `.stories.js` file, and we reference them here by id